2022-08-11 18:55:42,708 INFO    MainThread:345297 [internal.py:wandb_internal():90] W&B internal server running at pid: 345297, started at: 2022-08-11 18:55:42.703484
2022-08-11 18:55:42,711 INFO    WriterThread:345297 [datastore.py:open_for_write():75] open: /fsx/ganayu/code/SuperShaper/wandb/offline-run-20220811_185539-sv7kqns4/run-sv7kqns4.wandb
2022-08-11 18:55:42,712 DEBUG   HandlerThread:345297 [handler.py:handle_request():141] handle_request: run_start
2022-08-11 18:55:42,723 DEBUG   HandlerThread:345297 [meta.py:__init__():35] meta init
2022-08-11 18:55:42,723 DEBUG   HandlerThread:345297 [meta.py:__init__():49] meta init done
2022-08-11 18:55:42,723 DEBUG   HandlerThread:345297 [meta.py:probe():209] probe
2022-08-11 18:55:42,732 DEBUG   HandlerThread:345297 [meta.py:_setup_git():199] setup git
2022-08-11 18:55:42,816 DEBUG   HandlerThread:345297 [meta.py:_setup_git():206] setup git done
2022-08-11 18:55:42,816 DEBUG   HandlerThread:345297 [meta.py:_save_pip():53] save pip
2022-08-11 18:55:42,819 DEBUG   HandlerThread:345297 [meta.py:_save_pip():67] save pip done
2022-08-11 18:55:42,821 DEBUG   HandlerThread:345297 [meta.py:_save_conda():74] save conda
2022-08-11 18:55:51,715 WARNING MainThread:345297 [internal.py:is_dead():387] Internal process exiting, parent pid 345263 disappeared
2022-08-11 18:55:51,715 ERROR   MainThread:345297 [internal.py:wandb_internal():149] Internal process shutdown.
2022-08-11 18:55:51,715 INFO    WriterThread:345297 [datastore.py:close():279] close: /fsx/ganayu/code/SuperShaper/wandb/offline-run-20220811_185539-sv7kqns4/run-sv7kqns4.wandb
2022-08-11 18:55:51,721 INFO    SenderThread:345297 [sender.py:finish():1111] shutting down sender
2022-08-11 18:55:52,211 DEBUG   HandlerThread:345297 [meta.py:_save_conda():84] save conda done
2022-08-11 18:55:52,211 DEBUG   HandlerThread:345297 [meta.py:probe():247] probe done
2022-08-11 18:55:52,983 INFO    HandlerThread:345297 [handler.py:finish():806] shutting down handler
2022-08-11 18:55:52,984 INFO    MainThread:345297 [internal.py:handle_exit():80] Internal process exited
